docs/api/Client.md

```diff
@@ -48,6 +48,31 @@ import { Client } from 'undici'
 const client = new Client('http://localhost:3000')
 ```
 
+### Example - Custom connector
+
+This will allow you to perform some additional check on the socket that will be used for the next request.
+
+```js
+'use strict'
+import { Client, buildConnector } from 'undici'
```

Comment:
we should document `buildConnect()` somewhere else as well.

Reply:
Done!